### ISSUE TITLE
"Collection/set name" metadata not being populated.

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1638]
Originally reported on Google Code with ID 1643

```
Would like to see MCM populate the XBMC/KODI <set></set> tags from TMDB without using
collection/folder hierarchy. This was mentioned (below) a few weeks ago but I could
not find a bug report or feature request.

http://forums.mediacentermaster.com/viewtopic.php?f=4&t=10460
```

Reported by `edwardbackstrom` on 2015-02-07 17:35:34

**Comment by Happy2Play** [comment]
If MCM populated the "Collection/set name on Item metadata menu, this would populate <collection> tag in xml and <set> tag in nfo.

**Comment by mrchief** [comment]
I was wondering the same. I would like to keep folder structure flat (and not nested per collection) and have MCM populate `<set/>` in the nfo/xml file.

**Comment by psouza4** [comment]
Feature added in an upcoming release.

Metadata will need to be re-fetched when that release is available.

**Comment by psouza4** [comment]
Released in version 2.18 build 4518.858 (2.18.4518.858).
